pylint/checkers/utils.py

```diff
@@ -1522,3 +1522,19 @@ def get_iterating_dictionary_name(
         return node.iter.as_string()
 
     return None
+
+
+def get_subscript_const_value(node: astroid.Subscript) -> astroid.Const:
+    """
+    Returns the value (subscript.slice) of a Subscript node,
+    also supports python <3.9 windows where node.slice might be an Index
+    node
+    """
```

Comment:
Since this method will also raise an exception in some case, it would be good if you can add `:raises <ExceptionName>: ...` to the docstring. You can take a look at `utils.get_argument_from_call` how it's done there. (The parameter and return value don't need to be added though.)

Reply:
resolved in cd86ca0 -- had to add params/return details since it was caught by pylint as bad practice